changelog/unreleased/kong/bump_openssl.yml

```diff
@@ -0,0 +1,2 @@
+message: "Bumped OpenSSL to 3.2.2"
```

Comment:
Now, they require lib bump should explain why is this bump needed in changelog, like `Bumped xyz to 1.2.3 to fix an issue ...`